src/main/resources/migrations/00051_add_gateway_account_id_to_payout.sql

```diff
@@ -8,4 +8,4 @@ ALTER TABLE payout ADD COLUMN gateway_account_id VARCHAR(255);
 --changeset uk.gov.pay:add_index_on_gateway_account_id_to_payout_table
 CREATE INDEX payout_gateway_account_id_idx ON payout(gateway_account_id);
 
---rollback drop index payout_gateway_account_id_idx;
\ No newline at end of file
+--rollback drop index payout_gateway_account_id_idx;
```

Comment:
Not sure if liquibase will break as it calculates checksum for content and does it ignore if it sees `00051_add_gateway_account_id_to_payout.sql` as already migrated ?